flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/plan/util/AggregateUtil.scala

```diff
@@ -695,15 +699,43 @@ object AggregateUtil extends Enumeration {
     (propPos._1, propPos._2, propPos._3)
   }
 
-  def isRowtimeIndicatorType(fieldType: TypeInformation[_]): Boolean = {
-    TimeIndicatorTypeInfo.ROWTIME_INDICATOR == fieldType
+  def isRowtimeIndicatorType(fieldType: TypeInformation[_]): Boolean = fieldType match {
```

Comment:
What do you think about to move `isRowtimeIndicatorType`, `isProctimeIndicatorType`, `isTimeIntervalType` and `isRowIntervalType` to FlinkTypeFactory? It is wired that they appears in `AggregateUtil`.

Reply:
FlinkTypeFactory has already referenced InternalType, RelDataType, and provide convert conversion functions between these two types.  If we move `isRowtimeIndicatorType`,` isProctimeIndicatorType`, `isTimeIntervalType` and `isRowIntervalType` to FlinkTypeFactory, then FlinkTypeFactory will reference three types, which is not good.